wp-content/themes/citylimits/less/_navbar.less

```diff
@@ -0,0 +1,280 @@
+            padding-left: 25px;
+            border-left: 2px solid @grey2;
+            li {
+                &.menu-item {
+                    &:first-of-type {
+                        font-weight: bold;
+                        border-right: 2px solid @black;
+                    }
+                    a {
+                        font-size: 15px;
+                        color: @dark_grey;
+                        text-transform: uppercase;
+                        line-height: 15px;
+                        padding-left: 5px;
+                        padding-right: 5px;
+                        &:hover {
+                            background: none;
+                            color: @red;
+                        }
+                    }
+                }
+            }
+        }
+        @media (max-width: 999px) and (min-width: 769px) {
+            .nav {
+                li {
+                    &.menu-item {
+                        a {
+                            padding-left: 5px;
+                            padding-right: 5px;
+                            font-size: 14px;
+                        }
+                    }
+                }
+                padding-right: 10px;
+            }
+            .languages-nav {
+                padding-left: 5px;
+                li {
+                    &.menu-item {
+                        a {
+                            font-size: 14px;
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+#sticky-nav {
+    .container {
+        width: auto;
+        padding: 0;
+        display: flex;
+        justify-content: space-between;
+    }
+    #mobile-sticky-nav {
+        margin: 0;
+        border-right: 2px solid @grey2;
+        align-self: center;
+        float: left;
+        li {
+            a {
+                font-size: 15px;
+                font-weight: 500;
+                text-transform: uppercase;
+                padding-left: 15px;
+                padding-right: 15px;
+                line-height: normal;
+                &:hover {
+                    background: none;
+                    color: @red;
+                }
+            }
+        }
+    }
+    .nav-right {
+        float: right;
+    }
+    #header-extras {
+        li {
+            a.toggle-nav-bar {
+                line-height: 30px;
+                padding-top: 10px;
+                .bars {
+                    display: block;
+                    .icon-bar {
+                        margin-left: auto;
+                        margin-right: auto;
+                    }
+                }
+                span {
+                    font-size: 15px;
+                    font-weight: 500;
+                    text-transform: uppercase;
+                }
+                &:hover {
+                    color: @red;
+                    background: none;
+                }
+            }
+        }
+    }
+    &.open {
```

Comment:
If the mobile nav is open, and the display resizes, then the menu remains open, and the sticky nav gets weird:
![Screen Shot 2019-10-16 at 14 12 43 ](https://user-images.githubusercontent.com/1754187/66946500-11839800-f01f-11e9-954b-ef90ac69df20.png)

Reply:
Ah, gross. I guess we can just hide it after a certain width. Or did you have something else in mind, @benlk?

Reply:
That's probably easier than an on-resize handler that optionally closes the menu if it's open.